lib/hyrax/hirmeos/metrics_tracker.rb

```diff
@@ -29,7 +29,7 @@ def get_translator_work_id(uuid)
   end
 
   def resource_to_hirmeos_json(work)
-    Hyrax::Hirmeos::WorkFactory.for(resource: work)
+    Hyrax::Hirmeos::MetricsTracker.work_factory.for(resource: work)
```

Comment:
Could this be simplified:
```suggestion
    work_factory.for(resource: work)
```